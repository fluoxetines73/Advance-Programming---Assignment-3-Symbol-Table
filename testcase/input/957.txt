INSERT b string
ASSIGN b 'HelloWorld'
PRINT
BEGIN
BEGIN
INSERT b string random
ASSIGN c unimportant random
BEGIN
BEGIN
PRINT
END
LOOKUP b
ASSIGN b 456