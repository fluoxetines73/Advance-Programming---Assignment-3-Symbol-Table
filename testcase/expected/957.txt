Invalid: INSERT b string random